ThunkLibs/include/common/Host.h

```diff
@@ -68,6 +68,10 @@ struct fex_guest_function_ptr {
 
     template<typename Ret, typename... Args>
     fex_guest_function_ptr(Ret (*ptr)(Args...)) : value(reinterpret_cast<void*>(ptr)) {}
+
+    inline operator bool() const {
+      return value != nullptr;
```

Comment:
With this operator being introduced, the `[[maybe_unused]]` tag can be removed from `value`

Reply:
You right, removed it.